dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1068,4 +1061,65 @@ public final Response delete(@Context final HttpServletRequest  request,
         }
     }
 
-}
+    /**
+     * Copies container to the specified host
+     *
+     * @param request
+     * @param response
+     * @param id       id identifier to copy.
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
```

Comment:
We need to add the Swagger annotations

Reply:
I have updated the code as per discussion with @dsilvam so @jdotcms can you please review the last commit and share your feedback please